### PR TITLE
feat: Added verify email page

### PR DESCRIPTION
**Pull request by bbtgnn**
closes #614

### CODE REVIEW

File: contexts/AuthContext.tsx [hunk 1]
```diff
@@ -101,6 +111,7 @@ export const AuthProvider = ({ children, publicPage = false }: any) => {
       const name = getItem("authName") as string;
       const email = getItem("authEmail") as string;
       const publicKey = getItem("eddsaPublicKey") as string;
+      const isVerified = Boolean(getItem("authIsVerified"));
```
Comment (bbtgnn) [inline contexts/AuthContext.tsx:114]: C1

File: contexts/AuthContext.tsx [hunk 2]
```diff
@@ -77,15 +41,61 @@ export type User = {
   name: string;
   publicKey: string;
   privateKey: string;
+  isVerified: boolean;
```
Comment (bbtgnn) [inline contexts/AuthContext.tsx:44]: C0 – This key is added to store `isVerified` information

Comment (bbtgnn) [reply contexts/AuthContext.tsx:44]: [This is a sequence of comments, next ones are C1, C2, C3]

File: contexts/AuthContext.tsx [hunk 3]
```diff
@@ -109,6 +120,7 @@ export const AuthProvider = ({ children, publicPage = false }: any) => {
         name,
         privateKey,
         publicKey,
+        isVerified,
```
Comment (bbtgnn) [inline contexts/AuthContext.tsx:123]: C1
After the edit in C0, TS complains because `setUser` requires the `isVerified` key

**Review by puria — APPROVED** [review]

**Pull request merged**